### PR TITLE
Update remove-license-from-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
Tested and found that to have M365 defender license, you need to have the basic subscription intact. Added that to the Note.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3554

File: microsoft-365/admin/email/remove-license-from-shared-mailbox.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Shared mailboxes usually don't require a license. Follow these instructions to r
 > 1. The shared mailbox has more than 50 GB of storage in use.
 > 2. The shared mailbox uses in-place archiving.
 > 3. The shared mailbox is placed in litigation hold.
+> 4. The shared mailbox has Microsoft Defender License assigned.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/remove-license-from-shared-mailbox.md]:
```suggestion
> 4. The shared mailbox has a Microsoft Defender license assigned.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**